website/blog/2025-11-10-release-1.23.md

```diff
@@ -0,0 +1,380 @@
+- **Podman Network support**: Manage your Podman networks directly from Podman Desktop.
+
+---
+
+## Release details
+
+### Support for a managed configuration
+
+Podman Desktop now supports managed configuration. Users can add a “managed-by” configuration file to enforce specific settings. This allows system administrators to apply configurations from a global system file. With this release, administrators can deploy a managed settings file to a predefined system location, enabling centralized control over Podman Desktop configurations across an entire organization.
+
+Check out more details in this demo done by [@cdrage](https://github.com/cdrage):
+<ReactPlayer src='https://www.youtube.com/watch?v=HOFpDHMIleQ' controls={true} width="100%" height="500px"  />
+
+### Customizable columns and Dashboard sections
+
+<ThemedImage
+alt="Customizable Dashboard Sections"
+sources={{
+    light: require('./img/podman-desktop-release-1.23/DashboardSections-light.png').default,
+    dark: require('./img/podman-desktop-release-1.23/DashboardSections-dark.png').default,
+  }}
+/>
+
+<ThemedImage
+alt="Customizable Table Columns"
+sources={{
+    light: require('./img/podman-desktop-release-1.23/TableColumns-light.png').default,
+    dark: require('./img/podman-desktop-release-1.23/TableColumns-dark.png').default,
+  }}
+/>
+
+The new Layout Manager feature allows users to customize their dashboard sections and table columns to match their workflow preferences. Users can now reorder dashboard sections, show or hide specific table columns, and tailor their view to display only the most relevant information for their tasks.
+
+### Created Docker context for Podman machine
+
+<ThemedImage
+alt="Docker Context for Podman Machine"
+sources={{
+    light: require('./img/podman-desktop-release-1.23/DockerContextExtension-light.png').default,
+    dark: require('./img/podman-desktop-release-1.23/DockerContextExtension-dark.png').default,
+  }}
+/>
+
+Podman Desktop now automatically creates a Docker context for each registered Podman machine, improving interoperability with Docker tooling. Each context is prefixed with `podman-` and derived from the Podman machine name, allowing users to seamlessly switch between contexts using the standard `docker context use` command.
+
+### New features in Search Bar component
+
+<ThemedImage
+alt="Enhanced Search Bar with new features"
+sources={{
+    light: require('./img/podman-desktop-release-1.23/SearchBar-light.png').default,
+    dark: require('./img/podman-desktop-release-1.23/SearchBar-dark.png').default,
+  }}
+/>
+
+The search bar has been enhanced with intelligent text highlighting, quick navigation to application sections, and the ability to search for Podman resources (containers, images, pods, volumes) directly. The search bar now provides a command palette-like experience for faster navigation throughout the application.
```

Comment:
I would emphasis on the "We continued working on the search feature", do we have an issue we could link to get feedback? like an epic? might be nice to link it here

Reply:
Move the text above the image